servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -1049,10 +1032,6 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 
 				rp_uniform_set = _setup_render_pass_uniform_set(RENDER_LIST_ALPHA, p_render_data, radiance_texture, samplers, true);
 
-				if (rb_data.is_valid()) {
```

Comment:
Also removing this, as we've now merged our passes, we can just reuse the framebuffer we fetched at the start.